pages/style.css

```diff
@@ -0,0 +1,20 @@
+header {
```

Comment:
you have your css file in your pages directory. I would move your styles files into its own directory called styles